src/dune_rules/alias0.ml

```diff
@@ -25,4 +25,5 @@ let check = standard "check"
 let install = standard "install"
 let ocaml_index = standard "ocaml-index"
 let runtest = standard "runtest"
+let pkg_install = standard "pkg-install"
```

Comment:
It's incorrect to mark this alias as "standard" as it's not defined in every single directory.